scripts/release.ts

```diff
@@ -11,6 +11,7 @@ import { BuildOptions, getOptions } from './utils/options';
  * Runner for creating a release of Stencil
  * @param rootDir the root directory of the Stencil repository
  * @param args stringified arguments used to influence the release steps that are taken
+ * @returns a void promise
```

Comment:
Oh, are they enforcing this now for `void` and `Promise<void>`? That's disappointing 😢

Reply:
not for `void` and not, I think, for all async functions which actually return `Promse<void>`

for instance, this function it wouldn't ask for an annotation:

```ts
async function noReturn() {}
```

even though the return type is `Promise<void>`, but _this_ one it would ask for an annotation:

```ts
async function returnsPromise() {
  const promise = new Promise<void>(resolve => resolve());
  return promise;
}
```

Reply:
Thanks! I know there's been a regression or two in the library (or maybe some question around what they should require) in the past, so I just assumed this was more of the same in my original comment